Pipfile

```diff
@@ -18,6 +18,7 @@ alembic = "*"
 pytest = "==6.0.2"
 flask-bcrypt = "==0.7.1"
 flask-login = "==0.5.0"
+pandas = "*"
```

Comment:
Think it's better to give a specific version. For pandas, the current version is `1.1.3`.